raster/r.univar/stats.c

```diff
@@ -30,8 +30,8 @@ univar_stat *create_univar_stat_struct(int map_type, int n_perc)
     for (i = 0; i < n_zones; i++) {
 	stats[i].sum = 0.0;
 	stats[i].sumsq = 0.0;
-	stats[i].min = 0.0 / 0.0;	/* set to nan as default */
-	stats[i].max = 0.0 / 0.0;	/* set to nan as default */
+	stats[i].min = NAN;	/* set to nan as default */
```

Comment:
This change is not Cmake related and should be separated out

(into a new NAN related PR)